rdflib/store.py

```diff
@@ -169,10 +203,10 @@ def node_pickler(self):
         return self.__node_pickler
 
     # Database management methods
-    def create(self, configuration):
+    def create(self, configuration: Optional[Any]) -> None:
         self.dispatcher.dispatch(StoreCreatedEvent(configuration=configuration))
 
-    def open(self, configuration, create: bool = False):
+    def open(self, configuration: Any, create: bool = False) -> Optional[int]:
```

Comment:
I think the `configuration` param can be safely particularised as `str`, given the document text "Opens the store specified by the ***configuration string***"

Reply:
There is at least one store that takes a tuple and uses it to do something that cannot be done via a string:

https://github.com/RDFLib/rdflib/blob/3e2987bfed12bc7c76ad373097739029e3fc56e5/rdflib/plugins/stores/sparqlstore.py#L554-L568

https://github.com/RDFLib/rdflib/blob/3e2987bfed12bc7c76ad373097739029e3fc56e5/rdflib/plugins/stores/sparqlstore.py#L591-L609

Reply:
But fair enough, I think given docstrings say string I will make it a string.

Reply:
changed to string.